XIVSlothCombo/Combos/PvE/SAM.cs

```diff
@@ -215,26 +213,15 @@ internal class SAM_ST_SimpleMode : CustomCombo
             protected override uint Invoke(uint actionID, uint lastComboMove, float comboTime, byte level)
             {
                 SAMGauge? gauge = GetJobGauge<SAMGauge>();
-                bool oneSeal = OriginalHook(Iaijutsu) is Higanbana;
-                bool twoSeal = OriginalHook(Iaijutsu) is TenkaGoken or TendoGoken;
-                bool threeSeal = OriginalHook(Iaijutsu) is MidareSetsugekka or TendoSetsugekka;
+                bool oneSen = OriginalHook(Iaijutsu) is Higanbana;
+                bool twoSen = OriginalHook(Iaijutsu) is TenkaGoken or TendoGoken;
+                bool threeSen = OriginalHook(Iaijutsu) is MidareSetsugekka or TendoSetsugekka;
                 float enemyHP = GetTargetHPPercent();
                 bool trueNorthReady = TargetNeedsPositionals() && ActionReady(All.TrueNorth) && !HasEffect(All.Buffs.TrueNorth) && CanDelayedWeave(actionID);
-                float meikyostacks = GetBuffStacks(Buffs.MeikyoShisui);
-                
+                float GCD = GetCooldown(Hakaze).CooldownTotal;
+
                 if (actionID is Hakaze or Gyofu)
                 {
-                    if (IsEnabled(CustomComboPreset.SAM_Variant_Cure) &&
```

Comment:
Was the removal of Variant actions in Simple Mode by design? It's in the Adv Single, and both Simple/Adv AoE.

Reply:
Yes, however if it's recommended to keep them inside of Simple Mode as well I can add them back.